reposcan/repodata/repository_controller.py

```diff
@@ -80,19 +80,21 @@ def _check_cert_expiration_date(self, cert_name, cert):
             # Load certificate
             loaded_cert = crypto.load_certificate(crypto.FILETYPE_PEM, cert)
             # Get expiration date and parse it to datetime object
-            valid_to_dt = datetime.strptime(loaded_cert.get_notAfter(), "%Y%m%d%H%M%SZ")
+            valid_to_dt = datetime.strptime(loaded_cert.get_notAfter().decode("utf-8"), "%Y%m%d%H%M%SZ")
```

Comment:
Why we are using decode from utf-8 here?

Reply:
loaded_cert.get_notAfter() returns bytes (https://www.pyopenssl.org/en/stable/api/crypto.html#OpenSSL.crypto.X509.get_notAfter) but datetime.strptime() expects str, otherwise it crashes with TypeError

utf-8 doesn't have to be specified but is default, https://docs.python.org/3/library/stdtypes.html#bytes.decode